.github/workflows/labeler.yml

```diff
@@ -2,6 +2,10 @@ name: "Pull Request Labeler"
 on:
 - pull_request_target
 
+permissions:
+  contents: read
+  pull-requests: write
+
```

Comment:
This one, for example, already has a permissions block in the single job below. I do understand having it at a top-level for readability, but it'd be great if we could remove the other under `triage`.

Reply:
Either works.

I think I do prefer using a top-level permission default and then only overriding in workflows with multiple jobs that need distinct permissions as opposed to having workflows with single jobs where one has to look into the job for the permission.